code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -266,7 +266,7 @@
 
 /datum/gas/helium
 	id = "helium"
-	specific_heat = 15
+	specific_heat = 250
```

Comment:
lord allmighty

Reply:
Helium sees no use in the game, at least allow it as safe-ish coolant that you can use around the station that also makes voice funny?